### ISSUE TITLE
Translation support for title-post, title-page, title-cpt and metadesc

### ISSUE THREAD

**Issue by alexgarciab** [polylang/polylang#500]
## Prerequisites

- [X] This is not a usage question (Those should be directed to the [community supported forum](https://wordpress.org/support/plugin/polylang), unless this is a question about Polylang Pro in which case you should [use the helpdesk](https://polylang.pro/support/)).
- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest `master` branch of Polylang and the latest WordPress version.
- [X] This bug happens with only Polylang plugin active
- [X] This bug happens with a default WordPress theme active
- [X] I can reproduce this bug consistently

## Steps to reproduce the issue

1. Install PolyLang Pro
2. Trying to translate Wordpress SEO Strings
3. Not being able to translate them

## Expected behavior and actual behavior

Right now, when I am trying to translate SEO Strings in PolyLang Pro I ONLY see the following strings:

title-home-wpseo: %%sitename%% %%page%% %%sep%% %%sitedesc%%
title-author-wpseo: %%name%%, Author at %%sitename%% %%page%%
title-archive-wpseo: %%date%% %%page%% %%sep%% %%sitename%%
title-search-wpseo: You searched for %%searchphrase%% %%page%% %%sep%% %%sitename%%
title-404-wpseo: Page not found %%sep%% %%sitename%%

Why PolyLang Pro does not show up all the other Wordpress SEO related Strings? such as:

title-post: %%title%% %%page%% %%sep%% %%sitename%%
metadesc-post: %%excerpt%%
title-page: %%title%% %%page%% %%sep%% %%sitename%% 
metadesc-page: %%excerpt%%
title-cpt: %%title%% %%page%% %%sep%% %%sitename%% 
metadesc-cpt: %%excerpt%%

Basically I want to be able to translate "Post" titles and descriptions, "Pages" titles and descriptions and "Custom Post Types" titles and descriptions, to different language and right now I am not able to do it with PolyLang Pro.

**Comment by Chouby** [comment]
It works as expected for me. 

As a Polylang Pro customer, for such usage question, please use our helpdesk. Our support team would help you to solve your issue very quickly.

**Comment by alexgarciab** [comment]
Please, explain why this is the expected behavior? Why PolyLang is only showing those 5 strings and not the rest of the Yoast SEO strings?

**Comment by alexgarciab** [comment]
Solved with custom code
